administrator/language/de-DE/plg_system_webauthn.ini

```diff
@@ -23,16 +23,16 @@ PLG_SYSTEM_WEBAUTHN_ERR_NOT_DELETED="Der Authentifizierer konnte nicht entfernt
 PLG_SYSTEM_WEBAUTHN_ERR_NO_BROWSER_SUPPORT="Leider unterstützt der aktuelle Browser nicht den W3C-Web-Authentifizierungsstandard für passwortlose Anmeldungen oder diese Website wird nicht über HTTPS mit einem gültigen Zertifikat, das von einer Zertifizierungsstelle, welche der Browser vertraut, signiert ist, bereitgestellt. Bitte mit Benutzernamen und Passwort auf dieser Website anmelden."
 PLG_SYSTEM_WEBAUTHN_ERR_NO_STORED_CREDENTIAL="Die gespeicherten Anmeldedaten für diesen Login-Authentifizirer können nicht gefunden werden."
 PLG_SYSTEM_WEBAUTHN_ERR_USER_REMOVED="Der Benutzer für diesen Authentifizierer scheint auf dieser Website nicht mehr zu existieren."
-PLG_SYSTEM_WEBAUTHN_ERR_XHR_INITCREATE="Cannot get the authenticator registration information from your site."
-PLG_SYSTEM_WEBAUTHN_FIELD_ATTESTATION_SUPPORT_DESC="Only allow authenticators with verifiable cryptographic signatures to be used for WebAuthn logins. Strongly recommended for high security environments. Requires your site to be able to access <code>https://mds.fidoalliance.org/</code> directly, a writeable cache directory, the system temporary directory being writeable by PHP, and the OpenSSL extension. May prevent some cheaper, non-certified authenticators from working at all. Disabling it also prevents Joomla from identifying the make and model of the authenticator you are using (no icon will be displayed next to the Authenticator Name).<br/><strong>Pro tip:</strong> If you are behind a firewall you can place the data downloaded from <a href='https://mds.fidoalliance.org/' target='_blank'>the FIDO Alliance</a> into the file <code>administrator/cache/fido.jwt</code> for this feature to work properly."
+PLG_SYSTEM_WEBAUTHN_ERR_XHR_INITCREATE="Die Registrierungsinformationen des Authentifizierers können nicht von der Website abgerufen werden."
+PLG_SYSTEM_WEBAUTHN_FIELD_ATTESTATION_SUPPORT_DESC="Nur Authentifizierer mit überprüfbaren kryptografischen Signaturen für WebAuthn-Anmeldungen zulassen. Für Hochsicherheitsumgebungen dringend empfohlen. Erfordert, dass die Website direkt auf <code>https://mds.fidoalliance.org/</code> zugreifen kann, ein beschreibbares Cache-Verzeichnis, das temporäre Systemverzeichnis, welches von PHP beschreibbar ist, und die OpenSSL-Erweiterung. Kann einige billigere, nicht zertifizierte Authentifizierer daran hindern, überhaupt zu funktionieren. Das Deaktivieren hindert Joomla auch daran, Marke und Modell des verwendeten Authentifizierers zu identifizieren (neben dem Authentifikatornamen wird kein Symbol angezeigt).<br/><strong>Profi-Tipp:</strong> Wenn man sich hinter einer Firewall befindet, so können die von <a href='https://mds.fidoalliance.org/' target='_blank'>der FIDO-Allianz</a> heruntergeladenen Daten in die Datei <code>administrator/cache/fido.jwt</code> speichern, damit diese Funktion richtig funktioniert."
```

Comment:
Vorschlag:
PLG_SYSTEM_WEBAUTHN_FIELD_ATTESTATION_SUPPORT_DESC="Nur Authentifikatoren mit überprüfbaren kryptografischen Signaturen dürfen für WebAuthn-Anmeldungen verwendet werden. Wird für Umgebungen mit hohem Sicherheitsstandard dringend empfohlen. Voraussetzung ist, dass die Website direkt auf <code>https://mds.fidoalliance.org/</code> zugreifen kann, ein beschreibbares Cache-Verzeichnis und ein von PHP beschreibbares temporäres Systemverzeichnis verfügbar ist, sowie die OpenSSL-Erweiterung. Könnte manche billige, nicht zertifizierte Authentifikatoren daran hindern, dass sie richtig funktionieren. Die Deaktivierung verhindert auch, dass Joomla die Art und den Typ des verwendeten Authentifikators erkennt (neben dem Namen des Authentifikators wird kein Symbol angezeigt).<br/><strong>Profi-Tipp:</strong> Wenn man sich hinter einer Firewall befindet, kann die von <a href='https://mds.fidoalliance.org/' target='_blank'>der FIDO Allianz</a> heruntergeladenen Daten in der Datei <code>administrator/cache/fido.jwt</code> gespeichert werden, damit diese Funktion ordnungsgemäß funktioniert."

Reply:
@max123kl wenn du das direkt als suggestion einreichst, dann kann man das annehmen 👍 
Ist das kleine Symbol mit dem + und -

Reply:
```suggestion
PLG_SYSTEM_WEBAUTHN_FIELD_ATTESTATION_SUPPORT_DESC="Nur Authentifikatoren mit überprüfbaren kryptografischen Signaturen dürfen für WebAuthn-Anmeldungen verwendet werden. Wird für Umgebungen mit hohem Sicherheitsstandard dringend empfohlen. Voraussetzung ist, dass die Website direkt auf <code>https://mds.fidoalliance.org/</code> zugreifen kann, ein beschreibbares Cache-Verzeichnis und ein von PHP beschreibbares temporäres Systemverzeichnis verfügbar ist, sowie die OpenSSL-Erweiterung. Könnte manche billige, nicht zertifizierte Authentifikatoren daran hindern, dass sie richtig funktionieren. Die Deaktivierung verhindert auch, dass Joomla die Art und den Typ des verwendeten Authentifikators erkennt (neben dem Namen des Authentifikators wird kein Symbol angezeigt).<br/><strong>Profi-Tipp:</strong> Wenn man sich hinter einer Firewall befindet, kann die von <a href='https://mds.fidoalliance.org/' target='_blank'>der FIDO Allianz</a> heruntergeladenen Daten in der Datei <code>administrator/cache/fido.jwt</code> gespeichert werden, damit diese Funktion ordnungsgemäß funktioniert." 
```